Content.Shared/Doors/Systems/SharedDoorBoltSystem.cs

```diff
@@ -0,0 +1,54 @@
+using Content.Shared.Doors.Components;
+using Content.Shared.Popups;
+
+namespace Content.Shared.Doors.Systems;
+
+public abstract class SharedDoorBoltSystem : EntitySystem
+{
+
+    [Dependency] protected readonly SharedAppearanceSystem Appearance = default!;
+    [Dependency] protected readonly SharedAudioSystem Audio = default!;
+    [Dependency] protected readonly SharedPopupSystem Popup = default!;
+    public override void Initialize()
```

Comment:
i dont think Popup is used on the server so make it private, also newline

Reply:
Popup from server to client works because I have it in the serverside zombie code.